source/user-manual/api/configuration.rst

```diff
@@ -249,14 +249,16 @@ max_size
-+============+===============================================+===============+===================================================================================================================+
-| enabled    | yes, true, no, false                          | false         | Enable or disable log file rotation based on file size. This option will disable log file rotation based on time. |
-+------------+-----------------------------------------------+---------------+-------------------------------------------------------------------------------------------------------------------+
-| size       | Any positive number followed by a valid unit. | 1M            | Set a file size to trigger log rotation.                                                                          |
-|            | K/k for kilobytes, M/m for megabytes.         |               |                                                                                                                   |
-+------------+-----------------------------------------------+---------------+-------------------------------------------------------------------------------------------------------------------+
++------------+-----------------------------------------------+---------------+----------------------------------------------------------------------------------------------------------------------------+
+| Sub-fields | Allowed values                                | Default value | Description                                                                                                                |
++============+===============================================+===============+============================================================================================================================+
+| enabled    | yes, true, no, false                          | false         | Enable or disable log file rotation based on file size. This option will disable log file rotation based on time.          |
+|            |                                               |               | Rotation occurs after adding a new entry to the API log, triggering the selected method.                                   |
+|            |                                               |               | For instance, the time-based rotation will rotate after a new entry is added past midnight. Not at midnight automatically. |
++------------+-----------------------------------------------+---------------+----------------------------------------------------------------------------------------------------------------------------+
+| size       | Any positive number followed by a valid unit. | 1M            | Set a file size to trigger log rotation. This value must be equal to or greater than 1M.                                      |
```

Comment:
There are some extra spaces in this table:
![image](https://github.com/wazuh/wazuh-documentation/assets/23361101/3f5c2a86-8fe7-4c00-87b4-9fd7f3e20907)